std/os/child_process.zig

```diff
@@ -63,9 +64,11 @@ pub const ChildProcess = struct {
         PermissionDenied,
         InvalidUserId,
         ResourceLimitReached,
+        InvalidCharacter,
```

Comment:
these should not be errors that are possible for `ChildProcess.spawn` to emit.

Reply:
Why?  They are possible now due to it now using Utf8?  In saying that it should only return one of them, I believe InvalidCharacter, so I'll remove the other one.